lib/irb/init.rb

```diff
@@ -330,6 +330,10 @@ def IRB.parse_opts(argv: ::ARGV)
         @CONF[:USE_AUTOCOMPLETE] = true
       when "--noautocomplete"
         @CONF[:USE_AUTOCOMPLETE] = false
+      when "--regexp-completion"
```

Comment:
I'd like to use `completor` instead of `completion` so the term is consistent between flags and config names, which will be easier to remember.